src/murmur/Murmur.ice

```diff
@@ -49,8 +49,10 @@ module Murmur
 		int onlinesecs;
 		/** Average transmission rate in bytes per second over the last few seconds. */
 		int bytespersec;
-		/** Client version. Major version in upper 16 bits, followed by 8 bits of minor version and 8 bits of patchlevel. Version 1.2.3 = 0x010203. */
+		/** Legacy client version. Major version in upper 16 bits, followed by 8 bits of minor version and 8 bits of patchlevel. Version 1.2.3 = 0x010203. */
 		int version;
+		/** New client version. Each component has a size of 16 bits. */
+		long version2;
```

Comment:
```suggestion
		long version_v2;
```
Note, that we can't change `version` to `version_v1` though, as there might be downstream code relying on that variable name

Reply:
No this cant be done, because ICE does not want ``_`` in an identifier. I have tried that.

Reply:
:facepalm: 
Okay then